<!-- -*- mode: XML -*- -->
<launch>

    <!-- ################ map server for Path_planning  ################ -->
    <arg name="map_file" default="$(find turtlebot3_navigation)/maps/test_map.yaml"/>
    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)">
    <remap from="map" to="path_map"/>

    <!-- ################ start astar node ################ -->
    <arg name="namespace_one" default="robot_1"/>
    <node pkg="astar" type="astar" name="astar" output="screen">
        <param name="Euclidean" value="true"/>
        <param name="OccupyThresh" value="100"/>
        <param name="InflateRadius" value="0.126"/>
        <param name="rate" value="10"/>
        <param name="namespace" value="$(arg namespace_one)"/>
    </node>

</launch>
s